src/client/pages/ConnectPayment/sections/ConnectPayment.tsx

```diff
@@ -5,105 +5,131 @@ import { Market, useMarket } from 'components/utils/CurrentLocale'
 
 const SITEWRAPPER = 1300
-const BP = 800
-const MOBILE = 450
 
-const Background = styled('div')({
-  position: 'fixed',
-  top: 0,
-  right: 0,
-  bottom: 0,
-  left: 0,
-  backgroundColor: colorsV3.gray900,
-  zIndex: -1,
-})
+const Background = styled.div`
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: ${colorsV3.gray900};
+  z-index: -1;
+`
 
-const InnerWrapper = styled('div')({
-  display: 'flex',
-  flexDirection: 'row',
-  justifyContent: 'space-between',
-  alignItems: 'center',
-  maxWidth: SITEWRAPPER,
-  margin: 'auto',
-  paddingLeft: '10px',
-  paddingRight: '10px',
-  paddingTop: '25vh',
-  color: colorsV3.gray500,
-  [`@media (max-width: ${BP}px)`]: {
-    maxWidth: '100%',
-    flexDirection: 'column',
-    paddingTop: 40 + 70,
-  },
-})
+const InnerWrapper = styled.div`
+  display: flex;
+  width: 80%;
+  margin: 0 auto;
+  flex-direction: row;
+  align-items: flex-start;
+  max-width: ${SITEWRAPPER}px;
+  padding-left: 2rem;
+  padding-right: 2rem;
+  padding-top: 7.25rem;
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    padding-left: 1rem;
+    padding-right: 1rem;
+    padding-top: 6.75rem;
+    flex-direction: column-reverse;
+    width: 100%;
+  }
+`
 
-const TextColumn = styled('div')({
-  width: '60%',
-  paddingRight: 20,
-  paddingLeft: 20,
-  paddingBottom: 40,
+const TextColumn = styled.div`
+  width: 60%;
+  padding-bottom: 2.5rem;
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    width: 100%;
+  }
+`
 
-  [`@media (max-width: ${BP}px)`]: {
-    width: '100%',
-    textAlign: 'center',
+const ImageColumn = styled.div`
+  width: 40%;
+  margin-top: 0.5rem;
+  padding-left: 2rem;
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    width: 100%;
+    padding: 0;
+    text-align: center;
   },
-})
-
-const ImageColumn = styled('div')({
-  width: '40%',
-  paddingRight: 20,
-  paddingLeft: 20,
-  paddingBottom: 40,
+}
+`
 
-  [`@media (max-width: ${BP}px)`]: {
-    width: '100%',
-    textAlign: 'center',
-  },
-})
+const Header = styled.h1`
+  margin-top: 0;
+  margin-bottom: 2.25rem;
+  font-size: 3rem;
+  color: ${colorsV3.white};
+  ${MOBILE_SCREEN_MEDIA_QUERY} {
+    text-align: center;
+    margin-bottom: 2rem;
+    font-size: 2rem;
+    line-height: 2rem;
+  }
+`
 
-const Header = styled('h1')({
-  marginTop: '0px',
-  marginBottom: '30px',
-  fontSize: '56px',
-  lineHeight: '60px',
-  color: colorsV3.white,
-  [`@media (max-width: ${MOBILE}px)`]: {
-    fontSize: '44px',
-    lineHeight: '52px',
-  },
-})
+type HeaderPartProps = {
+  addWhiteSpace?: boolean
+}
+const HeaderPart = styled.span<HeaderPartProps>`
+  display: block;
+  font-family: ${fonts.FAVORIT};
+  font-weight: 400;
 
-const HeaderPart = styled('span')({
-  display: 'block',
-  fontFamily: fonts.FAVORIT,
-  fontWeight: 400,
-})
+  ${MOBILE_SCREEN_MEDIA_QUERY} {
+    display: initial;
+    ${(props) =>
+      props.addWhiteSpace &&
+      `
+      :before {
+        content:'\\00a0';
+      }`}
```

Comment:
This solution does the trick for sure! But I think it's easier to understand what the code does if just having a margin :) 
```suggestion
        margin-left: 0.375rem;
```